src/main/java/com/github/dcysteine/neicustomdiagram/generators/gregtech5/oreprocessing/GregTechOreProcessing.java

```diff
@@ -94,12 +96,15 @@ public DiagramGroup generate() {
             OTHER_ORE_PREFIXES
                     .forEach(prefix -> rawOres.addAll(GregTechOreDictUtil.getAllComponents(prefix, material)));
 
-            buildDiagram(matcherBuilder, rawOres);
+            ItemComponent trueRawOres = GregTechOreDictUtil.getAllComponents(OrePrefixes.rawOre, material).get(0);
```

Comment:
Will there ever be more than one raw ore item? If not, then consider using `getComponent` here instead, which will already be in `Optional` form.

(Actually, I think that works even if there's more than one, but I'm not sure)